wandb/integration/sb3/__init__.py

```diff
@@ -96,10 +103,18 @@ def _init_callback(self) -> None:
         wandb.config.update(d)
 
     def _on_step(self) -> bool:
-        if self.model_save_path is not None:
-            if self.n_calls % self.model_save_freq == 0:
-                self.model.save(self.path)
-                wandb.save(self.path)
-                if self.verbose > 1:
-                    print("Saving model checkpoint to", self.path)
+        if self.model_save_freq > 0:
+            if self.model_save_path is not None:
+                if self.n_calls % self.model_save_freq == 0:
+                    self.save_model()
         return True
+
+    def _on_training_end(self) -> None:
+        if self.model_save_path is not None:
+            self.save_model()
+
+    def save_model(self) -> None:
+        self.model.save(self.path)
+        wandb.save(self.path, base_path=self.model_save_path)
+        if self.verbose > 1:
+            print("Saving model checkpoint to", self.path)
```

Comment:
Use a logger here.

Reply:
Included logging in the latest changes, which looks something like

```
2021-07-21 10:49:09,086 INFO    MainThread:640427 [sb3.py:save_model():123] Saving model checkpoint to models/pynbeiz2/model.zip
```